app/assets/stylesheets/custom.css.scss

```diff
@@ -60,3 +60,17 @@ body {
   float: left;
   margin: 10px;
 }
+
+.item-title :hover {
+  color: #bfa24b;
+  text-decoration: none;
+}
+
+.category :hover {
+  color: #bfa24b;
```

Comment:
Color literals like `#bfa24b` should only be used in variable declarations; they should be referred to via variable everywhere else.